README.md

```diff
@@ -7,39 +7,77 @@ A collection of code snippets built with [Script Lab](github.com/OfficeDev/scrip
-2. Ensure you have a recent build of Node [6.10+] (`node -v`). Then install `yarn` as a global package `npm install yarn --global`.
-3. Run `yarn install` (similar to `npm install`, but better; and that's what is used by Travis, so best to have the same environment in both places)
-4. Create a snippet using [Script Lab](https://github.com/OfficeDev/script-lab/blob/master/README.md#what-is).  Ensure that the name and description are what you want them to be shown publicly.
-5. Choose the Share icon, and then choose **Copy to Clipboard**. 
-6. Open the folder where you want to store your code sample, and create a .yaml file. Paste your code sample into that file. Ensure that the code sample file names and folder names are in [`kebab-case`](http://wiki.c2.com/?KebabCase). To order folders and code samples:
-    - To order folders in a particular way, add a numeric prefix to the folder name (for example, "03-range"). The folder will be ordered sequentially in the list, and the prefix ("03-") will be removed.
-    - To order code samples in a particular folder, add **order: <#>** at the top of the code sample file(s). Code samples with order numbers will be sorted relative to the order specified.
-7. Stage the change:
-    1. Open the office-js-snippets project in Visual Studio Code.
-    2. Open the Source Control tab in VS Code. Your new YAML file will be listed in the **Source Control** pane just to the right of the tabs.
-    3. Right-click the file name and select **Stage Changes**. Leave VS Code open.
-10. Run `npm start`. If you received warnings, review the output to check what caused the build validation to fail, and fix as needed. As part of your troubleshooting, you can see what changes `npm start` made to the file:
-    1. In VS Code, you will see that Source Control pane now has two sections: **Staged Changes** and **Changes**, with your file listed under both.
-    2. Right-click the file name under **Changes** and select **Open changes**. Both versions of the file will be open in the editor so you can see the pending changes relative to the staged version. You may find that the script already substituted in required fields like `id` or `api_set` with reasonable defaults. 
-12. Re-run `npm start`, and fix errors, until the build succeeds. 
-13. Submit to the repo, and create a merge request into master.
+ > Note: In a few days, the repo admins will merge your snippet into the `prod` branch. It will then appear in **Samples** area of Script Lab. (It is in the **My Snippets** area as soon as you create it.) 
+
+25. Optionally, you can delete the branch you created from your fork and/or your local clone.
+
+#### Known errors and fixes in the build tool
+- An error saying that `name` has upper-case letters or other disallowed characters is *not* referring to the `name` property in the file. It is referring to the file name itself. You'll also get this error, if the file extension is not `.yaml`.
```

Comment:
Do you want to just fix the tool? :-)  I can show you how, if you're interested.  Will be available in the afternoon...